tests/run-pass/atomic.rs

```diff
@@ -85,3 +86,17 @@ fn atomic_fences() {
     compiler_fence(Acquire);
     compiler_fence(AcqRel);
 }
+
+fn weak_sometimes_fails() {
+    static ATOMIC: AtomicBool = AtomicBool::new(false);
```

Comment:
Any particular reason why this is not a local variable?

Reply:
Because I copied the other test 😅